bls/src/types/public_key.rs

```diff
@@ -19,6 +19,8 @@ pub struct PublicKey {
 }
 
 impl PublicKey {
+    pub const TRUSTED_SERIALIZATION_SIZE: usize = 570;
```

Comment:
Is this the same as `PUBLIC_KEY_SIZE`? If so I would prefer to name this `SIZE`, similar to `CompressedPublicKey::SIZE`.

```suggestion
    /// The size of a serialized PublicKey in bytes.
    pub const SIZE: usize = 570;
```